RELEASE-NOTES.md

```diff
@@ -1,3 +1,53 @@
+#Riak S2 (Riak CS) 2.1.1 Release Notes
+
+## General Information
+This is a bugfix release.
+
+## Bug Fixes
+
+* Remove blocks and a manifest after client failure. If a manifest was
+  left as writing and no overwrite happened after upload canceled,
+  then those partially uploaded blocks and manifest were to be left as
+  it was and occupies certain disk space. This is fixed by adding
+  additional error handling routine to move the manifest to GC bucket
+  by catching socket error. Partially uploaded blocks and manifest
+  will eventually be deleted by garbage
+  collection. ([#770](https://github.com/basho/riak_cs/issues/770) /
+  [PR#1280](https://github.com/basho/riak_cs/pull/1280)).
+* Remove `admin.secret` from `riak-cs.conf` or `stanchion.conf`. Both
+  Riak CS and Stanchion used them to find who was the administrator,
+  with `admin.key` and `admin.secret`. Writing down `admin.secret` in
+  non-encrypted form may risk the whole system of the secret being
+  stolen. Administrator is able to (1) list all users, (2)
+  disable/enable other users, (3) changing user accounts. Workaround
+  for this might be encrypting the partition that includes
+  `/etc/riak-cs`
+  directory. ([#1274](https://github.com/basho/riak_cs/issues/1274) /
+  [PR#1279](https://github.com/basho/riak_cs/issues/1279) /
+  [PR#108](https://github.com/basho/stanchion/pull/108)).
+* Forward-ported
+  [#1261](https://github.com/basho/riak_cs/issues/)/[PR#1263](https://github.com/basho/riak_cs/pull/1263)
+  as [PR#1266](https://github.com/basho/riak_cs/pull/1266).
+* Allow any bytes (including non-UTF8 ones) in List Objects response
+  XML. List Objects API had been failing in case where an object with
+  its name including non-utf8 character was included in the
```

Comment:
non-UTF8 and/or XML/1.0 incompatible

(This statement is somewhat redundant because XML/1.0 incompatibility includes non-UTF8, but I guess it's better for understandability)